protected/views/user/_form.php

```diff
@@ -56,19 +56,21 @@
       }
     }
 ?>
- 	
+
         <div class="container">
-              <section class="page-title-section">
+              <div class="section page-title-section">
                 <div class="page-title">
-                    <ol class="breadcrumb pull-right">
-                        <li><a href="/">Home</a></li>
-                        <li class="active">Personal details</li>
-                    </ol>
+                    <nav aria-label="breadcrumbs">
+											<ol class="breadcrumb pull-right">
+													<li><a href="/">Home</a></li>
+													<li class="active">Personal details</li>
+											</ol>
+										</nav>
                     <h4>Registration</h4>
                 </div>
-            </section>
+            </div>
         <div class="subsection" style="margin-bottom: 130px;">
```

Comment:
note: rest of changes of this file are a result of autoformatting, without code changes